src/webgl-utils.js

```diff
@@ -0,0 +1,181 @@
+/**
```

Comment:
Is this something that we can pull off of NPM instead? Or find something similar that is in NPM?

Reply:
Need to keep this as the npm version didn't quite work.